src/pages/Approvals/AddNewApprovalForm.tsx

```diff
@@ -0,0 +1,138 @@
+import { useForm } from "react-hook-form";
```

Comment:
What do you think in terms of a UI for having the approval page be on a person's page rather than a separate page?

For example, under the "add note" could be the "add approval" form?

![Screenshot from 2023-03-02 10-47-20](https://user-images.githubusercontent.com/829379/222478018-372aaa6c-a652-4330-8b41-76b032f3e5a7.png)

The membership/waiver/ff might need to be compressed a bit but I think the real estate is there. Maybe even make the emails / groups a two-column layout to compress the empty space there as well / instead.

That way, you remove the step of finding a person, and can see their rental history already in the panel while you're contemplating their approval.

Having a full list of approvals in a separate tab still has value (to see them all!) but it might make for a slightly easier UI to add the approval directly to a person.

That way, you can also show any active approvals directly on a person's page as well, rather than having to check the master list.

If we're implementing from scratch, we can make the UI flow a bit easier!

Reply:
Yes, definitely agree that we should display people's approval on their page, and add the ability to grant approval from there. 
I didn't include it in this PR as I wanted to initially match the existing features, and then build on top of it iteratively. For instance I'd like to also add a warning when desk workers are renting a restricted item when there is no approval.

My initial thought was to add a new tab before "Rental history", but I can explore including something in the person's summary.